packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -31,6 +31,39 @@ typedef PaintRangeValueIndicator = void Function(PaintingContext context, Offset
 ///
 /// Used to select a range from a range of values.
 ///
+/// {@tool dartpad --template=stateful_widget_scaffold}
+///
+/// This is what it looks like when run:
```

Comment:
I think we can leave this line (and the others) out. I think it's safe to assume that the developer will understand that the image represents what the sample app looks like.

Reply:
Done